documentation/markdown/accesories/aluminium-case/user_guide/user_guide.md

```diff
@@ -0,0 +1,106 @@
+<center>
+
+![Fin logo](../../../assets/balenaFin_logo.png)
+
+| **Document Type** | User Guide |
+| --- | --- |
+| **Product ID:** | TBD |
+| **Product Name** | Modular DIN-rail Case |
+| **Document Version** | 0.0.1 |
+| **Author** | Carlo Maria Curinga |
+| **State (Draft/Proposed/Approved)** | Draft |
+
+</center>
+
+## Revision history
+
+<center>
+
+| **Date (dd/mm/yyyy)** | **Version** | **Author** | **Description** |
+| --- | --- | --- | --- |
+| 14/01/2020 | 0.0.1 | Carlo Maria Curinga | First release |
+
+</center>
+
+<div class="page-break"></div>
+
+# 1. Introduction
+
+![Fin aluminium case](./pictures/assembled.jpg)
+
+The Aluminium case is the official "heavy duty" case for the balenaFin. It exposes the following ports to the outside :
+
+- 2 x USB
+- Ethernet (RJ45)
+- HDMI
+- Barrel jack power connector
+- 2 x External antenna mounting points
+
+# 2. What's in the box?
+
+1. Aluminium Case for balenaFin  
+    1.1 - bottom  
```

Comment:
Does the DIN rail clip come pre-assembled?

Reply:
yes